build.gradle

```diff
@@ -98,7 +98,7 @@ docker {
     files bootJar.archivePath
     buildArgs(['JAR_FILE': "${bootJar.archiveName}"])
 }
-dockerPrepare.dependsOn test
+dockerPrepare.dependsOn bootJar
```

Comment:
unrelated change, but when pushing docker it went wrong unless bootJar or assemble was called before.